exploration_maneuver_duration_secs: 18
exploration_maneuver_phases_duration_secs: 5
px4_loiter_radius: 0.5
px4/flight_speed: 3
offboard_enabled: true
laser_range: 9.5
laser_angle: 0.872665

odometry_error: 0

frontier/safety_margin: 3

geofence_min/x: -5
geofence_max/x: 15

geofence_min/y: -10
geofence_max/y: 10

geofence_min/z: 5
geofence_max/z: 15

path/max_time_secs: 120
path/max_cycles_waited_for_path: 50
path/safety_margin: 3

current_position_topic: "/ground_truth_to_tf/pose"

uav/laser_topic: "hokuyo_laser_scan"

